src/Features/tutorial.ts

```diff
@@ -0,0 +1,50 @@
+
+export function initTutorial(){
+    WA.player.getPlayerProperty('firstConnection').then((firstConnectionProperty) => {
+            if(firstConnectionProperty.propertyValue != "false"){
+                openTutorial();
+                WA.player.setPlayerProperty({
+                    propertyName: 'firstConnection',
+                    propertyValue: false
+                })
+            }
+        }
+    )
+}
+
+export function openTutorial() {
+    if (/Mobi|Android/i.test(navigator.userAgent)) {
+        // Creates tutorial iFrame for mobile devices
+        WA.room.website.create({
+            name: 'tutorial',
+            url: '/tutorial.html',
+            position: {
+                x: 5,
+                y: 75,
+                width: 375,
+                height: screen.height,
+            },
```

Comment:
The position is expressed in absoluted coordinates in the map (not the screen).
So it needs to be relative to the position of the player. Otherwise, on big maps where the player does not start in the top left corner, the tutorial will not be visible.

Also, ideally, we would need a way to center an iframe relative to the viewport and not to the map.